third_party/hip/hip_stub.h.inc

```diff
@@ -54,6 +54,7 @@ enum hipError_t {
   hipErrorOperatingSystem = 304,
   hipErrorInvalidHandle = 400,
   hipErrorInvalidResourceHandle = 400,
+  hipErrorIllegalState = 401,
```

Comment:
Do we also need to update printing for these error enums?

Reply:
Actually, I will let you do this in a separate change.

Reply:
> 

Probably not. However, they are added in ROCm 5.0.